lib/alarm.dart

```diff
@@ -9,6 +9,7 @@ import 'package:alarm/src/android_alarm.dart';
 import 'package:alarm/service/notification.dart';
 import 'package:alarm/service/storage.dart';
 import 'package:flutter/foundation.dart';
+import 'package:flutter_local_notifications/flutter_local_notifications.dart';
```

Comment:
I would prefer if the `flutter_local_notifications` code is only called from the `AlarmNotification` service. Would that be possible ?

Reply:
Yes, it is totally possible.

I imported flutter_local_notifications package to lib/alarm only for the NotificationResponse type in onSelectNotification. Any code from flutter_local_notifications is not used.

I thought about circular imports. lib/alarm uses lib/services/notification and that's why i placed onSelectNotification in lib/alarm.
But i see that lib/services/notification also uses lib/alarm (alarmPrint) and [the circular dependencies are allowed for dart](https://stackoverflow.com/questions/65324492/dart-circular-imports)

I replaced the callback and added a new commit.

Reply:
Awesome thank you.